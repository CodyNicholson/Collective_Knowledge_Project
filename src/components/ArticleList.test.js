//This file was begun using code from Link.react.test.js, from the Jest Testing React Apps Guide
//https://facebook.github.io/jest/docs/en/tutorial-react.html
import React from 'react';
import ArticleList from './ArticleList';
import renderer from 'react-test-renderer';

test('ArticleList component test', () => {
  const component = renderer.create(
	<ArticleList>
	</ArticleList>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});