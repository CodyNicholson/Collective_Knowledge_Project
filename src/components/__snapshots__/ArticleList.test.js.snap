// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags module test 1`] = `
<div
  className="article-preview"
>
  Loading...
</div>
`;
